lib/internal/fs/streams.js

```diff
@@ -183,10 +187,16 @@ ReadStream.prototype._read = function(n) {
       // Now that we know how much data we have actually read, re-wind the
       // 'used' field if we can, and otherwise allow the remainder of our
       // reservation to be used as a new pool later.
-      if (start + toRead === thisPool.used && thisPool === pool)
-        thisPool.used += bytesRead - toRead;
-      else if (toRead - bytesRead > kMinPoolSpace)
-        poolFragments.push(thisPool.slice(start + bytesRead, start + toRead));
+      if (start + toRead === thisPool.used && thisPool === pool) {
+        const alignedOffset = thisPool.used + bytesRead - toRead;
+        thisPool.used = roundUpToMultipleOf8(alignedOffset);
+      } else {
+        const alignedEnd = (start + toRead) & ~7; // round down to multiple of 8
```

Comment:
One more micro-nit: comments should be capitalized and punctuated (and normally have two spaces before the `//` - I expect the linter will complain.)